lib/livebook_web/live/session_live/input_cell_settings_component.ex

```diff
@@ -28,7 +28,7 @@ defmodule LivebookWeb.SessionLive.InputCellSettingsComponent do
         <div class="flex flex-col space-y-6">
           <div>
             <div class="input-label">Type</div>
-            <%= render_select("type", [number: "Number", password: "Password", text: "Text", textarea: "Textarea", url: "URL"], @type) %>
+            <%= render_select("type", [number: "Number", password: "Password", text: "Text", textarea: "Textarea", url: "URL", color: "Color"], @type) %>
```

Comment:
We should probably move this to a private function and keep it alphabetically ordered:

    defp input_types do
      [
        color: "Color",
        number: "Number",
        password: "Password",
        text: "Text",
        textarea: "Textarea",
        url: "URL"
      ]
    end